README.md

```diff
@@ -10,4 +10,30 @@ What if...
 
 ...it had better transfer and caching properties than actual websites?
 
-...you could host the whole database in a plain-old browser tab?
\ No newline at end of file
+...you could host the whole database in a plain-old browser tab?
+
+## Contributing
```

Comment:
I'll echo my sentiment that I don't think Radicle should be the low bar to contributing. It solves a problem, specifically that GitHub is a single point of failure and authority. But, where contributing is concerned it seems inferior on almost every metric compared to GitHub (and also most self-hosted options).

Reply:
I'll paste my bottom line comment from our other conversation:

> I'll meet you half way on Radicle if you are willing to meet me half way on low-barriers to entry / participation / contribution.

Reply:
I don't mind it in the README, but it would be nice if basic contributions took a more conventional approach.